x11-misc/clipmenu/clipmenu-6.0.1.ebuild

```diff
@@ -17,12 +17,12 @@ IUSE=""
 RDEPEND="
 	x11-misc/clipnotify
 	x11-misc/dmenu
-	x11-misc/xsel
-"
+	x11-misc/xsel"
```

Comment:
Moving the double quote to the end of the previous line seems like an unrelated change. Could you revert it, please?